utils/config_policy.go

```diff
@@ -6,76 +6,199 @@ import (
 	"io"
 	"os"
 	"path/filepath"
-	"slices"
 	"strconv"
+	"strings"
+	"time"
 
 	"github.com/PlakarKorp/kloset/locate"
 	"go.yaml.in/yaml/v3"
-	"gopkg.in/ini.v1"
 )
 
-type ConfigHandler interface {
-	ValidateKeyVal(key, val string) error
-	ValidateEntry(map[string]string) error
+type policiesConfig struct {
+	Version  string                           `yaml:"version"`
+	Policies map[string]*locate.LocateOptions `yaml:"policies"`
 }
 
-type policiesConfig struct {
-	Version  string                       `yaml:"version"`
-	Policies map[string]map[string]string `yaml:"policies"`
-}
-
-func (c *policiesConfig) ValidateKeyVal(key, value string) error {
-	if !slices.Contains([]string{
-		"keep-minutes",
-		"keep-hours",
-		"keep-days",
-		"keep-weeks",
-		"keep-months",
-		"keep-years",
-		"keep-per-minute",
-		"keep-per-hour",
-		"keep-per-day",
-		"keep-per-week",
-		"keep-per-month",
-		"keep-per-year",
-	}, key) {
-		return fmt.Errorf("invalid key")
-	}
+func (c *policiesConfig) Has(name string) bool {
+	_, ok := c.Policies[name]
+	return ok
+}
 
+func (c *policiesConfig) Add(name string) {
+	c.Policies[name] = &locate.LocateOptions{}
+}
+
+func (c *policiesConfig) setInt(value string, p *int) error {
 	i, err := strconv.Atoi(value)
 	if err != nil {
 		return fmt.Errorf("invalid value: %w", err)
 	}
 	if i < 0 {
 		return fmt.Errorf("negative value")
 	}
+	*p = i
+	return nil
+}
 
+func (c *policiesConfig) setBool(value string, p *bool) error {
+	if value == "true" || value == "yes" {
+		*p = true
+	} else if value == "false" || value == "no" {
+		*p = false
+	} else {
+		return fmt.Errorf("invalid value")
+	}
```

Comment:
I don't particularly link it, but since we're using it, can you switch this to `strconv.ParseBool` for consistency with the rest of the codebase?